casepro/msgs/tests.py

```diff
@@ -571,7 +571,9 @@ def test_search(self):
         # log in as a non-administrator
         self.login(self.user1)
 
-        response = self.url_get('unicef', url, {})
+        # check that appropriate number of queries are executed
+        with self.assertNumQueries(28):
```

Comment:
Hmm seems high. Can you check adding another FAQ (temporarily) and verify that this number doesn't increase? If it does then something is not quite right.